html/semantics/scripting-1/the-script-element/nomodule-set-dynamically-on-external-classic-scripts.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Dynamically inserted classic script with noModule set to false must not load</title>
+<link rel="author" title="Yusuke Suzuki" href="mailto:utatane.tea@gmail.com">
+<link rel="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<!-- Load this script synchronously to ensure test cases below can load it in 200ms -->
+<script src="resources/set-window-executed.js"></script>
+</head>
+<body>
+<script>
+
+promise_test(function () {
+    window.executed = false;
+    let loaded = false;
+    let errored = false;
+    return new Promise((resolve) => {
+        const element = document.createElement("script");
+        element.src = 'resources/set-window-executed.js';
+        document.body.appendChild(element);
+        element.onload = () => { loaded = true; }
+        element.onerror = () => { errored = true; }
+        setTimeout(resolve, 200);
+    }).then(() => {
+        assert_equals(window.executed, true);
+        assert_equals(loaded, true);
+        assert_equals(errored, false);
+    });
+}, 'An external classic script element dynamically inserted after noModule was set to false must run.');
+
+promise_test(() => {
+    window.executed = false;
+    let loaded = false;
+    let errored = false;
+    return new Promise((resolve) => {
+        const element = document.createElement("script");
+        element.noModule = true;
+        element.src = 'resources/set-window-executed.js';
+        document.body.appendChild(element);
+        element.onload = () => { loaded = true; }
+        element.onerror = () => { errored = true; }
+        setTimeout(resolve, 200);
+    }).then(() => {
+        assert_equals(window.executed, false);
```

Comment:
Doesn't the first test set `window.executed` to true? Maybe they should use different properties or be separate test files?